WordPress/gutenberg
# [RNMobile] - Tooltip for template selection buttons

`Gutenberg Mobile PR` -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/2216
`WordPress iOS PR` -> https://github.com/wordpress-mobile/WordPress-iOS/pull/14045
`WordPress Android PR` -> https://github.com/wordpress-mobile/WordPress-Android/pull/11807

## Description
This PR closes https://github.com/wordpress-mobile/gutenberg-mobile/issues/2169 

Adds a Tooltip to point to the template selection buttons. The component itself has a simple logic and it is attached to the Picker component. If we need another tooltip within the editor we could use this as a base and make it reusable. 

## How has this been tested?

<details><summary>Steps to test</summary>

- Open the main apps PR (iOS/Android).
- Download and install the build.
- Open the app and go to Pages.
- Tap on the add a new page button.
- **Expect** to see the templates picker with the tooltip on top.
- Scroll through the template options.
- **Expect** the tooltip to stay visible.
- Tap on the title of the page.
- **Expect** to see the tooltip fade out.
- Close the app.
- Open the app and go to Pages.
- Tap on the add a new page button.
- **Expect** to see the templates picker **without** the tooltip.

</details>

## Screenshots 

<details><summary>UX Interaction</summary>

iOS | Android
-|-
<img src="https://user-images.githubusercontent.com/4885740/81066610-a8417480-8edd-11ea-96f4-dc2feb51fe6c.gif" width="250" /> | <img src="https://user-images.githubusercontent.com/4885740/81066620-aa0b3800-8edd-11ea-8e8a-2402df181949.gif" width="250" />

</details>

<details><summary>iOS Screenshots</summary>

Tooltip | Tooltip Dark mode
-|-
<img src="https://user-images.githubusercontent.com/4885740/81061270-534d3080-8ed4-11ea-9905-3a24027b4948.png" width="250" /> | <img src="https://user-images.githubusercontent.com/4885740/81061283-5811e480-8ed4-11ea-99a5-f9ef6c2aa924.png" width="250" />

</details>

<details><summary>Android Screenshots</summary>

Tooltip | Tooltip Dark mode
-|-
<img src="https://user-images.githubusercontent.com/4885740/81061448-aa530580-8ed4-11ea-95f7-a64ec4589974.png" width="250" /> | <img src="https://user-images.githubusercontent.com/4885740/81061446-a921d880-8ed4-11ea-91e9-55f2553b853e.png" width="250" />

</details>

## Types of changes





## Checklist:
